site/_docs_1_0/annotations.md

```diff
@@ -1,11 +1,11 @@
-# Annotations
+---
+title: Annotation Reference
+layout: page
+---
 
-Annotations are used in Ingress Controllers to configure features that are not
-covered by the Kubernetes Ingress API.
+Annotations are used in Ingress Controllers to configure features that are not covered by the Kubernetes Ingress API.
 
-Some of the features that have been historically configured via annotations are
-supported as first-class features in Contour's [IngressRoute
-API](ingressroute.md), which provides a more robust configuration interface over
+Some of the features that have been historically configured via annotations are supported as first-class features in Contour's [IngressRoute API](ingressroute.md), which provides a more robust configuration interface over
```

Comment:
Does this link work? I expect it should be `{% link Ingressroute.md %}` or something?

Reply:
link does work, but it only seems to take absolute references. fixed.